conans/client/conf/config_installer.py

```diff
@@ -38,21 +38,21 @@ def _handle_profiles(source_folder, target_folder, output):
             relative_path = ""
         for f in files:
             profile = os.path.join(relative_path, f)
-            output.info("    Installing profile %s" % profile)
+            output.info("  - %s" % profile)
             shutil.copy(os.path.join(root, f), os.path.join(target_folder, profile))
 
 
 def _process_git_repo(repo_url, client_cache, output, tmp_folder, verify_ssl, args=None):
-    output.info("Trying to clone repo  %s" % repo_url)
+    output.info("Trying to clone repo: %s" % repo_url)
 
     with tools.chdir(tmp_folder):
         try:
             args = args or ""
-            subprocess.check_output('git -c http.sslVerify=%s %s clone "%s" config' % (verify_ssl, args, repo_url),
-                                    shell=True)
-            output.info("Repo cloned")
+            subprocess.check_output('git clone -c http.sslVerify=%s %s "%s" config'
```

Comment:
Can we use the `Git` wrapper here?

Reply:
I think it is not possible as you need to pass arguments to the git clone command and the Git class is designed to be used in the SCM approach

Reply:
The Git class is only used in what we call the _SCM feature_, but it is not tight to it, All the particularities of the _SCM feature_  should be in module `conans.model.scm`, not in the wrappers around `Git` or `SVN`.